extensions/ql-vscode/src/view/model-editor/ModelTypeTextbox.tsx

```diff
@@ -0,0 +1,52 @@
+import type { ChangeEvent } from "react";
+import { useCallback, useEffect, useState } from "react";
+import type {
+  ModeledMethod,
+  TypeModeledMethod,
+} from "../../model-editor/modeled-method";
+import { VSCodeTextField } from "@vscode/webview-ui-toolkit/react";
+import { useDebounceCallback } from "../common/useDebounceCallback";
+
+type Props = {
+  modeledMethod: TypeModeledMethod;
+  typeInfo: "path" | "relatedTypeName";
+  onChange: (modeledMethod: ModeledMethod) => void;
+
+  "aria-label"?: "Path" | "Related type name";
```

Comment:
I think this can just be a `string`, we don't really care about this value:

```suggestion
  "aria-label"?: string;
```

Reply:
Thanks!